HOSTING.md

```diff
@@ -0,0 +1,65 @@
+# Hosting the Rich Text Extension
```

Comment:
We have a change to host this extension rich-text.extensions.content.amplience.net, which will be released soon. Might be best to hold off merging this down until his hosting is in place.